frontend/src/index.tsx

```diff
@@ -275,6 +275,14 @@ const AuthenticationRoleRouter = () => {
 	const isAuthLoading = authRole === AuthRole.LOADING
 	const isLoggedIn = authRole === AuthRole.AUTHENTICATED
 
+	useEffect(() => {
```

Comment:
Force redirect to sign_in if auth_mode is password but user does not have 'passwordToken'